sysdig-probe-loader

```
@@ -320,6 +320,8 @@ build_kernel_probe() {
 				echo "* Found ${ko_file}; trying to insmod"
 				if insmod $ko_file > /dev/null 2>&1; then
 					echo "  ${PROBE_NAME}.${ext} found and loaded in dkms"
+					echo "* Tentatively running dkms install for ${PACKAGE_NAME}, all errors will be ignored"
```

```suggest
					echo "* Tentatively running dkms install for ${PACKAGE_NAME}"
```

---

@deepskyblue86 how do you like [`69a0bcf` (#160)](https://github.com/draios/probe-builder/pull/160/commits/69a0bcf0117fd8c87342a437ae6502658060e060) ?

---

BTW, the intention (in both the old and new incarnation) was to highlight the fact that such errors are ignored by the script (like they've always been) and therefore the user should not be alarmed.